hydra-main/src/main/java/com/addthis/hydra/query/aggregate/MeshSourceAggregator.java

```diff
@@ -133,12 +133,36 @@ void stopSources(String message) {
         }
     }
 
-    boolean tryActivateSource(QueryTaskSourceOption option) {
-        return option.tryActivate(meshy, queryOptions);
+    /**
+     * Attemps to active one of the options of the given QueryTaskSource.
+     *
+     * @param taskSource the QueryTaskSource to activate
+     * @return {@code true} if one of the options is activated; {@code false} if none of the
+     *         options can be/is activated
+     */
```

Comment:
how much of this javadoc can be inferred from the method name, parameters, and return type?